GVFS/GVFS.UnitTests/Service/Mac/MacServiceTests.cs

```diff
@@ -49,6 +51,22 @@ public void ServiceStartTriggersAutoMountForCurrentUser()
             repoRegistry.VerifyAll();
         }
 
+        [TestCase]
+        public void ServiceHandlesEnablePrjfsRequest()
+        {
+            string expectedServiceResponse = "TRequestResponse|{\"State\":1,\"ErrorMessage\":null}";
+            Mock<NamedPipeServer.Connection> connectionMock = new Mock<NamedPipeServer.Connection>(MockBehavior.Strict, null, this.tracer, null);

```

Comment:
What are the 2 `null`s you are passing here?  Will you use named parameters for them?

Reply:
As discussed the `Mock` constructors don't allow this. Added comments instead of parameter labels.

```
public Mock();
public Mock(MockBehavior behavior);
public Mock(MockBehavior behavior, params object[] args);
```